### ISSUE TITLE
[💡 FEATURE] - Personnalize Dark Mode

### ISSUE THREAD

**Issue by Floooowk** [alexkim205/G-Desktop-Suite#103]
**Is your feature request related to a problem? Please describe.**
When displaying a document (gsheet, gdoc ...), give the user the ability to chose between white and dark mode for the background of the page.
So, it will be possible to keep the page background in white and the UI in dark mode.

**Comment by dragonBall561** [comment]
Yes, you can change it. On the menubar, go to view -> Toggle Dark Mode. You can also use CTRL+T (Windows, Linux). If on windows, you can also go to `C:\Users\YOU\AppData\Roaming\G Desktop Suite` and open `config.json` in Notepad. Then set `theme` to `dark` or `light`
